python/hail/tests.py

```diff
@@ -37,6 +37,88 @@ def convert_struct_to_dict(x):
     else:
         return x
 
+def create_all_values_datasets():
+    all_values = hl.struct(
+        f32=hl.float32(3.14),
+        i64=hl.int64(-9),
+        m=hl.null(hl.tfloat64),
+        astruct=hl.struct(a=5, b=5.5),
+        aset=hl.set(['foo', 'bar', 'baz']),
+        d=hl.dict({hl.array(['a', 'b']): 0.5, hl.array(['x', 'y', 'z']): 0.3}),
+        i=hl.interval(
+            hl.locus('1', 999),
+            hl.locus('1', 1001)),
+        c=hl.call(0, 1),
+        t=hl.tuple([hl.call(1, 2, phased=True), 'foo']))
+
+    def prefix(s, p):
+        return hl.struct(**{p + k: s[k] for k in s})
+
+    all_values_table = (hl.utils.range_table(5, n_partitions=3)
+                        .annotate_globals(**prefix(all_values, 'global_'))
+                        .annotate(**all_values)
+                        .cache())
+
+    all_values_matrix_table = (hl.utils.range_matrix_table(3, 2, n_partitions=2)
+                               .annotate_globals(**prefix(all_values, 'global_'))
+                               .annotate_rows(**prefix(all_values, 'row_'))
+                               .annotate_cols(**prefix(all_values, 'col_'))
+                               .annotate_entries(**prefix(all_values, 'entry_'))
+                               .cache())
+    return all_values_table, all_values_matrix_table
+
+def create_backward_compatability_test_files():
+    import os
```

Comment:
Is this just run when needed? I don't see an actual call to this function.

Reply:
Correct, I ran it once in this version to produce the files in `src/test/resources/backward_compatability`.